src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1562,13 +1562,18 @@ def process_remove_identity_namespace(cmd, namespace):
 
 
 def process_gallery_image_version_namespace(cmd, namespace):
-    TargetRegion = cmd.get_models('TargetRegion')
+    TargetRegion, EncryptionImages, OSDiskImageEncryption, DataDiskImageEncryption = cmd.get_models(
+        'TargetRegion', 'EncryptionImages', 'OSDiskImageEncryption', 'DataDiskImageEncryption')
```

Comment:
do not need to add `operation_group` for it ?

Reply:
`operation_group` is `gallery_image_versions` here